starknet_py/net/account/account.py

```diff
@@ -100,10 +100,6 @@ def address(self) -> int:
     def client(self) -> Client:
         return self._client
 
-    @property
-    def supported_transaction_version(self) -> int:
-        return 1
-
```

Comment:
Actually, let's keep this property here and in `BaseAccount` and add a proper deprecation warning and note in docstring.

Reply:
Also include a note about this deprecation in migration guide "deprecations" section.